src/screens/RegisterEnsSheet.js

```diff
@@ -25,6 +26,54 @@ const Container = styled.View`
   flex: 1;
 `;
 
+const AvailabilityIndicator = props => {
```

Comment:
Should this component be in a separate file? perhaps we could have an ENS registration folder? `src/components/ens-registration/AvailabilityIndicator.js`?